include/nccl-headers/neuron/net.h

```diff
@@ -55,5 +55,6 @@ typedef void (*ncclDebugLogger_t)(ncclDebugLogLevel level, unsigned long flags,
 
 #include "net_v4.h"
 #include "net_v5.h"
+#include "net_v6.h"
```

Comment:
I'm confused.  Why bother adding both v5 and v6 in one go?

Reply:
1. It takes like one day to be able to merge an approved PR (tests need to run + last time, tests failed for whatever unrelated reason).
2. Reviewing multi-commit PR in github is somewhat easy.
3. Posting two CRs independently at the same time would require another round of rebasing, not?

Reply:
I think the question (or at least my question) is, what's the advantage of having the v5 API at all? It seems you're diving right into the v6 API. Is there any version of NCCOM that will use the v5 API?

Reply:
This split is what we came up with when we discussed that internally. Are there strong opinions against it?

Reply:
Unless you foresee any version of NCCOM actually using the v5 API, I would skip it and move directly to v6. Why have an extra API version that won't be used?

Reply:
After refactoring PR now contain three commits
1. Refactoring (moving v4 API into separate file)
2. Commit introducing v5 with (a) "non-blocking" connection establishment API and implementation behind it and (b) interface for RMA operations and properties about new operation and flat whether operations are supported (no RMA operation implementation, yet)
3. Commit adding RMA operations.